wemake_python_styleguide/violations/base.py

```diff
@@ -35,14 +35,14 @@
 Deprecating a violation
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-When you want to mark some violation as depracated,
+When you want to mark some violation as deprecated,
```

Comment:
Looks like you have previous changes in your PR 😢

Reply:
My mistake, sorry about that.